packages/cms/pages/settings/roles-and-permissions/roles-and-permissions.tsx

```diff
@@ -0,0 +1,344 @@
+            <EuiFieldText
+              onChange={event =>
+                setCreateRoleForm({
+                  ...createRoleForm,
+                  name: event.target.value
+                })
+              }
+              name="rolename"
+              fullWidth
+            />
+          </EuiFormRow>
+
+          <EuiFormRow label="Description" fullWidth>
+            <EuiTextArea
+              onChange={event =>
+                setCreateRoleForm({
+                  ...createRoleForm,
+                  description: event.target.value
+                })
+              }
+              fullWidth
+            />
+          </EuiFormRow>
+          <EuiSpacer size="xl" />
+          <FlyoutAccordion
+            createRoleForm={{
+              createRoleForm,
+              setCreateRoleForm
+            }}
+            allPermissions={allPermissions}
+          />
+        </EuiForm>
+      </EuiFlyoutBody>
+      <EuiFlyoutFooter>
+        <EuiFlexGroup justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty iconType="cross" flush="left" onClick={closeFlyout}>
+              Cancel
+            </EuiButtonEmpty>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButton onClick={closeFlyout} fill>
+              Create Role
+            </EuiButton>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      </EuiFlyoutFooter>
+    </EuiFlyout>
+  )
+}
+
+const RolesTable: React.FC = () => {
+  const [createRoleForm, setCreateRoleForm] = useState<CreateRoleForm>({
+    name: '',
+    description: '',
+    adminPermissions: []
+  })
+  const [allPermissions, setAllPermissions] = useState<AdminPermission[]>([])
+  const [adminRoles, setAdminRoles] = useState<any>([])
+  const [loading, setLoading] = useState(true)
+  const [isFlyoutOpen, setIsFlyoutOpen] = useState(false)
+
+  const columns = [
+    {
+      field: 'role',
+      name: 'role '
+    },
+    {
+      field: 'members',
+      name: 'members'
+    }
+  ]
+
+  useEffect(() => {
+    const fetchAdminRoles = async () => {
+      const [response] = await window.Tensei.api.get('admin-roles')
+
+      setAdminRoles(response?.data.data)
+
+      setLoading(false)
+    }
+    fetchAdminRoles()
+  }, [])
+
+  async function fetchPermissions() {
+    const [response] = await window.Tensei.api.get<{
+      data: AdminPermission[]
+    }>('admin-permissions')
+    if (response !== null) {
+      setAllPermissions(response.data.data)
+    }
+  }
+
+  useEffect(() => {
+    fetchPermissions()
+  }, [])
+
+  const renderedItems = adminRoles.map((item: any) => {
+    const numberOfMembers = adminRoles.filter(
+      (memberItem: any) => item.name === memberItem.name
+    )
+    const member = numberOfMembers.length > 1 ? 'members' : 'member'
+    return {
+      ...item,
+      role: item.name,
+      members: `${numberOfMembers.length} ${member}`
+    }
+  })
+
+  return (
+    <>
+      <TableHeading>
+        <div>
```

Comment:
When displaying text, be sure to use `EuiText` or `EuiTitle`. In this case, please use a title. The size of the title should be the same as the title of the profile page.